apps/twitch-bot/server/server.go

```diff
@@ -42,6 +43,22 @@ func (s *SenchabotAPIServer) GetTwitchBotConfig(ctx context.Context, twitchChann
 	return configData, nil
 }
 
+func (s *SenchabotAPIServer) CheckConfig(ctx context.Context, twitchChannelId string, configKey string, configValue string) bool {
+	configData, err := s.backend.GetTwitchBotConfig(ctx, twitchChannelId, configKey)
+	if err != nil {
+		fmt.Println(err.Error())
+		return false
+	}
+
+	if configData != nil {
+		if configData.Value == configValue {
+			return true
+		}
+	}
+
```

Comment:
I think this block might more short. example = `configData != nil && configData.Value == configValue`